app/containers/modals/tx-fees-top-up.js

```diff
@@ -7,7 +7,7 @@ import { BtcField } from "@containers/fields/btc-address";
 
 const TxFeesModal = ({ hide }) => (
   <Modal
-    title="Top Up"
+    title="Add BTC"
```

Comment:
Minor suggestion: "Add BTC for transaction fees" may be even more informative as a header

Reply:
This is done.